biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -1199,16 +1203,20 @@ public void report(PrintStream out) {
 					out.println("Id    Levl State Modified      Location");
 
 					for (int i = 0; i < bundles.length; i++) {
+						String lastModified = "<>";
 						String loc = bundles[i].getLocation();
-						loc = loc.replaceAll("\\w+:", "");
-						File f = new File(loc);
+						Path p = URIUtil.fromURI(loc);
+						if (p != null && Files.exists(p)) {
+							long time = Files.getLastModifiedTime(p)
```

Comment:
What is this `time` variable used for? It seems to be dead code.

Reply:
Debugging relic. Removed.